lib/src/components/Sections/index.vue

```diff
@@ -1690,62 +1695,129 @@ export default {
         }
       });
 
-      const token = this.$cookies.get("sections-auth-token");
-      const header = {
-        token,
-      };
-      const config = {
-        headers: sectionHeader(header),
-      };
+      let integrityCheck = true
+      if (sections.length > 0) {
+        this.types.forEach(type => {
+          if (type.fields && Array.isArray(type.fields) && type.fields.length > 0) {
+            sections.forEach(section => {
+              if (section.name === type.name) {
+                if (Array.isArray(section.options)) {
+                  type.fields.forEach(field => {
+                    section.options.forEach(option => {
+                      if (Object.keys(option).includes(field.name)) {
+                        if(option[field.name] && (Array.isArray(option[field.name]) || typeof option[field.name] === 'object')) {
+                          if (Array.isArray(option[field.name])) {
+                            if ((!option[field.name][0].media_id || !option[field.name][0].url) && option[field.name].length !== 0) {
+                              integrityCheck = false
+                              this.loading = false;
+                              this.showToast(
+                                "Error saving your changes",
```

Comment:
the error titles should also be translated

Reply:
For now, after replacing the toast component for nuxt sections, it does not show any title which is why all titles were kept without translations